index.html

```diff
@@ -21,37 +21,49 @@ <h4 class="range-input">Set Range</h4>
+              <input id="max-range-input" type="number" class="range-input game-input" name="maxRange"/>
               <p class="error magenta"><img src="error-icon.svg"/> Must be less than Min!</p>
             </section>
             <section class="content-box">
-              <input type="button" class="game-button range-button" id="update-button" disabled value="UPDATE" />
+              <input type="button" class="range-button game-button" id="update-button" disabled value="UPDATE" />
             </section>
           </div>
         </section>
         <section id="input-for-game" class="game-box">
           <form class="" action="index.html" method="post">
             <p>The Current Range is <span class="bold underlined range-view-js"><strong>1</strong></span> to <span class="bold underlined range-view-js"><strong>100</strong></span></p>
             <h4>Challenger 1</h4>
-            <label for="player1">Name</label>
-            <input class="game-input player" id="player1"  type="text" name="player">
-            <label for="guess1">Guess</label>
-            <input class="game-input guess" id="guess1" type="number" name="player">
+            <section class="challenger-box">
+              <section class="content-box">
+                <label for="player1">Name</label>
+                <input class="game-input player" id="player1"  type="text" name="player" />
+              </section>
+              <section class="content-box">
+                <label class="guess" for="guess1">Guess</label>
+                <input class="game-input guess" id="guess1" type="number" name="player" />
+              </section>
+            </section>
             <h4>Challenger 2</h4>
-            <label for="player2">Name</label>
-            <input class="game-input player" id="player2" type="text" name="player">
-            <label for="guess2">Guess</label>
-            <input class="game-input guess" id="guess2" type="number" name="player">
+            <section class="challenger-box">
+              <section class="content-box">
+                <label for="player2">Name</label>
+                <input class="game-input player" id="player2" type="text" name="player">
+              </section>
+              <section class="content-box">
+                <label class="guess" for="guess2">Guess</label>
+                <input class="game-input guess" id="guess2" type="number" name="player">
+              </section>
+            </section>
             <section class="button-box">
-              <label for="submit-button" class="hide">SUBMIT GUESS</label>
+              <label for="submit-button" class="hide button-text">SUBMIT GUESS</label>
               <input type="button" id="submit-button" disabled class="play-input game-button" name="" value="SUBMIT GUESS">
-              <label for="reset-button" class="hide">RESET GAME</label>
+              <label for="reset-button" class="hide button-text">RESET GAME</label>
```

Comment:
Why did we add the class button-text to labels that are hidden with the hide class they aren't being shown on the page

Reply:
As well as I don't see button-text in the css?

Reply:
Yeah, I was trying to style the button text but didn't get it to work quite yet.  I'll just cut that out for now.